controllers/controllers.go

```diff
@@ -27,6 +30,14 @@ import (
 	"github.com/openshift/windows-machine-config-operator/version"
 )
 
+var (
+	// CSIConfiguredLabel indicates that this Node was configured by a version of WMCO with CSI storage. When upgrading this
+	// Node CSI storage migration does not have to be accounted for.
```

Comment:
Not sure what I was trying to say with this comment. Will fix next revision :sweat_smile: